### PR TITLE
Add block storage support

### PR DESCRIPTION
**Pull request by aajdinov**
Added block storages support (POCO classes, client endpoint, unit tests, and documentation).

### CODE REVIEW

File: OneAndOne.UnitTests/BlockStorage/BlockStoragesTest.cs [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        [TestMethod]
+        public void ShowBlockStorage()
+        {
+            var result = client.BlockStorages.Show(blockStorage.Id);
+
+            Assert.IsNotNull(result);
+            Assert.IsNotNull(result.Id);
+        }
+
+        //[TestMethod]
```
Comment (jasmingacic) [inline OneAndOne.UnitTests/BlockStorage/BlockStoragesTest.cs]: Are these commented lines necessary?

File: OneAndOne.UnitTests/BlockStorage/BlockStoragesTest.cs [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+            var result = client.BlockStorages.CreateServerBlockStorage(serverBlockStorage, blockStorage.Id);
+
+            Assert.IsNotNull(result);
+            Assert.IsNotNull(result.Id);
+
+            Config.WaitBlockStorageReady(result.Id);
+        }
+
+        //API returns 405 NOT ALLOWED
```
Comment (jasmingacic) [inline OneAndOne.UnitTests/BlockStorage/BlockStoragesTest.cs]: Same as before

**Comment by edevenport** [comment]
@aajdinov Please fix the merge conflicts.

**Comment by aajdinov** [comment]
@edevenport I've merged the master branch into this one and fixed the conflicts.

**Pull request merged**